File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands): 
    answer = []
    
    for command in commands:
        
        i = command[0] -1
        j = command[1] 
        k = command[2] -1
        

        print(i,j,k)
        
        new_array = array[i:j]
        
        #print(new_array)
        new_array.sort()
        print(new_array)
        answer.append(new_array[k])
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int xx = 0;
        for(int[] cmd : commands){
            int i = cmd[0];
            int j = cmd[1];
            int k = cmd[2];
            
            int size = j - i + 1;
            
            int[] arr = new int[size];
            
            int x = 0;
            for(int idx = i-1; idx<= j-1; idx++ ){
                arr[x] = array[idx];
                x++;
            }
            Arrays.sort(arr);
            answer[xx] = arr[k-1];
            xx++;
        }
        
        return answer;
    }
}